pages/wiki/styles.module.scss

```
@@ -0,0 +1,123 @@
+@import '@/styles/import.scss';
+
+.container {
+  @include Size(100%, 100vh);
+  @include Flex(column, inherit, center);
+  max-width: 100%;
+  margin: 0 auto;
+  position: relative;
+  box-sizing: border-box;
+  padding-top: 78px;
+
+  .wiki-main {
+    display: grid;
+    grid-template-columns: minmax(60px, 1fr) minmax(0, 860px) 80px 320px 200px;
+    width: 100%;
+    max-width: 1920px;
+    position: relative;
+
+    .wiki-padding {
+      grid-column: 1;
+      grid-row: 1;
+      width: 100%;
+      max-width: 460px;
+      overflow: hidden;
+    }
+
+    .wiki-header {
+      grid-column: 2;
+      grid-row: 1;
+      margin-bottom: 20px;
+      width: 100%;
+      box-sizing: border-box;
+    }
+
+    .wiki-article {
+      grid-column: 2;
+      grid-row: 2;
+      width: 100%;
+      box-sizing: border-box;
+    }
+
+    .wiki-aside {
+      grid-column: 4;
+      grid-row: 1 / span 2;
+      width: 320px;
+      position: relative;
+    }
+  }
+}
+
+@media screen and (max-width: 1199px) {
+  .container {
+    padding: 0 60px;
+    padding-top: 60px;
```

```suggestion
@media screen and (max-width: $tablet-max-width) {
  .container {
    padding: 0 60px;
    padding-top: 60px;
```
`common.scss`에 보시면 태블릿과 모바일 responsive 변수가 있어요~!